docs/data-sources/packer_image.md

```diff
@@ -56,6 +56,7 @@ output "packer-registry-ubuntu" {
 - `organization_id` (String) The ID of the organization this HCP Packer registry is located in.
 - `packer_run_uuid` (String) UUID of this build.
 - `project_id` (String) The ID of the project this HCP Packer registry is located in.
+- `revoke_at` (String) The revocation time of this build
```

Comment:
What is the zero value for this field when there is no revocation time? 

I think it could be helpful to indicate what format the output would be in and what it is set to if no revocation time is available.
```
"The revocation time of this build. This field will be null for any build that has not been revoked or scheduled for revocation. ",
```

Reply:
Oh yeah, that's very important! I will add that information